### DISCUSSION TITLE
Need clarity about group creation: Is public group (anyone can join group themselves) functianality available?

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Lern/Community#165] [Documentation]
Hello team!!!

We have a requirement with following functionalities
Admins should have the flexibility to configure these groups in two ways
1. as invite-only,(same as working now on portal)
2. as **Open groups: open for any user to request to join the group**.
3. For Open groups, **discussions from forum of those should be visible to all users**.(to achieve public discussions in forum)

are abouve tings possible?
Also I was checking to create group. I found **membershipType**  in request body. I wanted to know what happens if we set **membershipType as moderated**?? how we can use this?

Please help us here

@sknirmalkar89, @reshmi-nair, @AmiableAnil,

**Comment by Hari-stackroute** [comment]
@paritshivani As of now option 2 and 3 are not available.
And of now membershipType value is saving as either of one invite-only/moderated, no special privileges is given as of now based on membershipType.

**Reply by Jayaprakash8887** [reply]
@paritshivani Kindly mark the thread as answered if there are no further concerns

**Reply by mohitga** [reply]
@paritshivani will be able to update the thread as answered, if the issue is solved?
